generative_model_training_all_labels/01_generative_model_dag_all_labels.ipynb

```diff
@@ -0,0 +1,1734 @@
+{
```

Comment:
<p> </p><p>Double checking, this database is local/not connected to the internet right? Just want to make sure since the username and password are saved with the code</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/snorkeling-full-text/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Yup the database is local. People will have to hack penn's security to get there; however, there isn't any sensitive data in the database anyways. All public that anyone can access, but good question.